File: 2022/01/0102.py
f = open("01-input.txt", "r")

max: list[int] = [0,0,0];
counter = 0;
for x in f:
    if x.strip() == "":
        if any(i < counter for i in max):
            max[max.index(min(max))] = counter;
        counter = 0;
        continue;
    
    counter += int(x);
print(sum(max));

f.close();

File: 2022/01/0102.kt
import java.io.File

fun main() {
    var max = arrayOf(0, 0, 0)
    var counter: Int = 0
    var cont: Boolean = false
    
    File("input.txt").forEachLine { it ->
        if (it.isEmpty()) {
            if (max.any{it2 -> it2 < counter}) {
                max[max.indexOf(max.min())] = counter
            }
            counter = 0
            cont = true
        }
        if (!cont) {
            counter += it.toInt()
        }
        cont = false
    }
    println(max.sum())
}